players/utils.py

```diff
@@ -58,8 +70,8 @@ def change_picking_person_for_serpentine_draft(
 
 
 def change_picking_person_for_fixed_draft(
-    team, next_profile_index, group_profiles_order
-):
+    team: Team, next_profile_index: int, group_profiles_order: List[str]
```

Comment:
O tu wystarczy `list[str]` i bez importa na górze.

Ofc mówimy tylko o pythonie 3.9+ ale skoro tak masz z `tuple` wyżej, to znaczy, że takiego używasz.